platforms/extensibility/plugin-development/src/main/java/org/gradle/plugin/devel/tasks/PluginUnderTestMetadata.java

```diff
@@ -45,24 +45,21 @@ public abstract class PluginUnderTestMetadata extends DefaultTask {
 
     public static final String IMPLEMENTATION_CLASSPATH_PROP_KEY = "implementation-classpath";
     public static final String METADATA_FILE_NAME = "plugin-under-test-metadata.properties";
-    private final ConfigurableFileCollection pluginClasspath = getProject().files();
-    private final DirectoryProperty outputDirectory = getProject().getObjects().directoryProperty();
 
     /**
      * The code under test. Defaults to {@code sourceSets.main.runtimeClasspath}.
+     *
+     * Note: this is not a @Classpath since the ablosute paths are written to the metadata file.
      */
-    @Classpath
```

Comment:
❓ Is it ok to change `@Classpath` to `@InputFiles` here?

Reply:
This seems correct to me.